src/django/api/views/facility/facilities_view_set.py

```diff
@@ -220,6 +222,12 @@ def list(self, request):
                 ]
             }
         """
+        page_serializer = FacilityListPageParameterSerializer(
```

Comment:
@vladsha-dev Why you not combine a page validation logic into FacilityQueryParamsSerializer?

Reply:
I didn’t do that because the serializer is used in other places, and it could break the logic there.

Reply:
Got it!